### DISCUSSION TITLE
How do I set the correct context for GitHub.community?

### DISCUSSION THREAD

**Discussion by elgohr** [community/community#22953] [Enterprise]
<p>Hi,<br>
maybe the topic is a little misleading, sadly I didn’t find a better one.<br>
So my account is part of a GitHub Enterprise organization, where SAML login is configured and required.<br>
When I want to use this forum (GitHub.community), it requires me to authenticate with SAML. In fact my actions here have nothing to do with that particular organization. How can I make this clear, so that I don’t need to SAML authenticate here?</p>

**Comment by nyahbhinghiPrincess** [comment]
<p>Hi <a class="mention" href="/elgohr">@elgohr</a>,</p>
<p>Thank you for reaching out with this question and we are so glad to have you here in the community!  At the moment, the sign-in credentials to the GitHub Community Forum are tied to that of the sign-in for your <a href="http://GitHub.com" rel="nofollow noopener">GitHub.com</a> account. </p>
<p>We’re always working to improve GitHub and the GitHub Community Forum, so I’ve logged your feedback in our internal feature request list.  Presently, in order to have a separate login method, or separate profile in the community forum, you would need to be signing in with an account that is independent of your Enterprise Organization.</p>

**Comment by elgohr** [comment]
<p>Hi,<br>
thank you for the feedback.<br>
As Github Terms &amp; Conditions require one account per natural person, I don’t think that this is a solution…</p>
<p>Edit: Second feedback: when you have special characters, like ‘&amp;’ in a post and you edit the post, it isn’t resolved to the character again, but to the HTML representation. Are you building this board from scratch?</p>

**Comment by nyahbhinghiPrincess** [comment]
<p>Hi <a class="mention" href="/elgohr">@elgohr</a>,</p>
<p>You are correct on ToS as mentioned here in our help doc on <a href="https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/merging-multiple-user-accounts" rel="nofollow noopener">Merging multiple user accounts</a> ; I apologize for the misunderstanding.  My example was just to imply that having a separate login method into the GitHub Community forum is not presently possible as it is linked the <a href="http://GitHub.com" rel="nofollow noopener">GitHub.com</a> account.  I hope that helps to clear up the explanation used.</p>
<p>We appreciate the feedback on editing your post with the ampersand and other special characters.  I’ve taken your insight and passed it along to the appropriate teams!</p>

**Comment by elgohr** [comment, marked as answer]
<p>Seems to be fixed, as only the repositories require a special login now. Thank you!</p>
